servicecontrol/capacity-and-planning.md

```diff
@@ -1,79 +1,80 @@
  * Failed messages *never* expire and are retained indefinitely in the ServiceControl database.
 
 
 #### Performance
 
-From a performance perspective, ServiceControl is similar to a database installation. It requires a significant amount of disk and network I/O due to processing of audit, error and monitoring messages. Each of these message-processing operations requires disk I/O. The higher the message throughput of an environment, the higher the required disk I/O.
+From a performance perspective, ServiceControl is similar to a database installation. It requires a significant amount of disk and network I/O due to process audit, error and monitoring messages. Each of these message-processing operations requires disk I/O. The higher the message throughput of an environment, the higher the required disk I/O.
 
-For this reason, it is necessary to store ServiceControl data on a disk with the lowest possible latency for I/O operations. Indexes require continuous updating and will require significant RAM to allow those indexes to be kept in memory. Processing of indexes that cannot be stored fully in RAM will result in a higher likelihood of those indexes being stale. Since messages are added to a full-text search, it is also necessary to make sure the CPU will not become a bottleneck in updating indexes.
+For this reason, it is best to store ServiceControl data on a disk with low latency for I/O operations. Indexes are continuously updated and keeping them in memory requires significant RAM. Indexes that cannot be stored fully in RAM are more likely to be stale. Since messages are added to full-text search, it is also necessary to ensure the CPU has sufficient capacity for updating indexes.
 
 For more details, see [Hardware Considerations](servicecontrol-instances/hardware.md).
 
+
 ### Accessing data and audited messages
 
 
-#### Alternate audit and error queues
+#### Forwarding queues
 
-ServiceControl can be configured to forward any consumed messages into alternate queues. A copy of any message consumed by ServiceControl is available from these alternate queues.
+ServiceControl may be configured to forward consumed messages to other queues for further consumption by third party systems.
 
 See also: [Forwarding Queues](errorlog-auditlog-behavior.md).
 
 
-#### Query the ServiceControl HTTP API
+#### HTTP API
 
-This provides a JSON stream of audited and error messages (headers, body, and context) that can be imported into another database.
+The ServiceControl HTTP API provides a JSON stream of audited and error messages (headers, body, and context) that can be imported into another database.
 
-NOTE: ServiceControl HTTP API is subject to changes and enhancements that may not be backward compatible. Use of this HTTP API is discouraged by third parties at this time.
+NOTE: The ServiceControl HTTP API is subject to changes and enhancements that may not be backward compatible. Use of the HTTP API by third parties is discouraged at this time.
```

Comment:
I'm not sure why this section is in this document.

Reply:
Likely the result of a question or support case from a customer. I'm 👍 to remove it though.

Reply:
I think it's valuable information, but it seems completely out of place in this document. Anyway, I'll merge this editorial review and pursue this point separately. Thank for the review.